README.md

```diff
@@ -30,45 +30,45 @@ It is not possible to run a full Taskcluster deployment directly from this repos
-    * [Purge-Cache Service](services/purge-cache)
-    * [Web-Server Service](services/web-server)
-    * [Treeherder Service](services/treeherder)
-    * [Secrets Service](services/secrets)
-    * [Notify Service](services/notify)
-    * [Github Service](services/github)
-    * [Queue Service](services/queue)
-    * [Login Service](services/login)
-    * [Index Service](services/index)
-    * [Hooks Service](services/hooks)
-    * [Auth Service](services/auth)
-* [Taskcluster Web](ui)
-    * [ui/src/components/DateDistance](ui/src/components/DateDistance)
-    * [ui/src/components/StatusLabel](ui/src/components/StatusLabel)
-    * [ui/src/components/SpeedDial](ui/src/components/SpeedDial)
-    * [ui/src/components/Snackbar](ui/src/components/Snackbar)
-    * [ui/src/components/Markdown](ui/src/components/Markdown)
-    * [ui/src/components/Search](ui/src/components/Search)
+* [Infrastructure](infrastructure#readme)
+    * [References](infrastructure/references#readme)
+    * [Terraform](infrastructure/terraform#readme)
+    * [Taskcluster Builder](infrastructure/builder#readme)
+* [Libraries](libraries#readme)
+    * [Typed-Env-Config Library](libraries/typed-env-config#readme)
+    * [References Library](libraries/references#readme)
+    * [Validate Library](libraries/validate#readme)
+    * [Testing Library](libraries/testing#readme)
+    * [Iterate Library](libraries/iterate#readme)
+    * [Monitor Library](libraries/monitor#readme)
+    * [Scopes Library](libraries/scopes#readme)
+    * [Taskcluster Client](libraries/client#readme)
+    * [Loader Library](libraries/loader#readme)
+    * [Azure Library](libraries/azure#readme)
+    * [Pulse Library](libraries/pulse#readme)
+    * [Docs Library](libraries/docs#readme)
+    * [API Library](libraries/api#readme)
+    * [App Library](libraries/app#readme)
+* [Services](services#readme)
+    * [Built-In Workers Service](services/built-in-workers#readme)
+    * [Purge-Cache Service](services/purge-cache#readme)
+    * [Web-Server Service](services/web-server#readme)
+    * [Treeherder Service](services/treeherder#readme)
+    * [Secrets Service](services/secrets#readme)
+    * [Notify Service](services/notify#readme)
+    * [Github Service](services/github#readme)
+    * [Queue Service](services/queue#readme)
+    * [Login Service](services/login#readme)
+    * [Index Service](services/index#readme)
+    * [Hooks Service](services/hooks#readme)
+    * [Auth Service](services/auth#readme)
+* [Taskcluster Web](ui#readme)
+    * [ui/src/components/DateDistance](ui/src/components/DateDistance#readme)
+    * [ui/src/components/StatusLabel](ui/src/components/StatusLabel#readme)
+    * [ui/src/components/SpeedDial](ui/src/components/SpeedDial#readme)
+    * [ui/src/components/Snackbar](ui/src/components/Snackbar#readme)
+    * [ui/src/components/Markdown](ui/src/components/Markdown#readme)
+    * [ui/src/components/Search](ui/src/components/Search#readme)
```

Comment:
These README's are used by the tc-web's [styleguide](https://taskcluster-web-styleguide.netlify.com/). We should probably remove these links from here 😅 We could take care of that in a separate PR though.

Reply:
That should be fine.  There's a few `.filter(..)`'s in the generation code -- this would just mean adding one more.  Feel free!